### PR TITLE
Fix debendency conflicts

### LINKED ISSUE #58: Bump dependencies with vulnerabilities
Dependency org.yaml:snakeyaml:1.26 is vulnerable
CVE-2022-25857 7.5 Uncontrolled Resource Consumption vulnerability pending CVSS allocation
CVE-2022-38752 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38749 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38750 5.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38751 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
Results powered by Checkmarx(c)

Dependency ch.qos.logback:logback-classic:1.2.3 is vulnerable
CVE-2021-42550 6.6 Deserialization of Untrusted Data vulnerability pending CVSS allocation
Results powered by Checkmarx(c)

### LINKED ISSUE #60: Add dependency management

### PR DESCRIPTION
**Pull request by Elishtar**
I haven't forgotten about:
- [x] Tests
- [x] Changelog
- [ ] Documentation
- [x] Commit messages comply with the [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)
- [x] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:

Closes #60 
Closes #58

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,6 +55,27 @@
...
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-annotations</artifactId>
+                <version>2.14.0</version>
+            </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>2.0.0</version>
+            </dependency>
```
Comment (ArtDu) [inline pom.xml:69]: I think `snakeyaml` should be added here too

Comment (ArtDu) [reply pom.xml:69]: Can you bump version to remove CVE alerts besides ?https://github.com/tarantool/cartridge-java-testcontainers/issues/58

```
Dependency org.yaml:snakeyaml:1.26 is vulnerable
CVE-2022-25857 7.5 Uncontrolled Resource Consumption vulnerability pending CVSS allocation
CVE-2022-38752 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38749 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38750 5.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38751 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
Results powered by Checkmarx(c)
```

Comment (Elishtar) [reply pom.xml:69]: Hm, let's try to use 1.33

Comment (Elishtar) [reply pom.xml:69]: Done

**Review by ArtDu — APPROVED** [review]

**Review by ArtDu — CHANGES_REQUESTED** [review]
I forgot, add some words in changelog

**Comment by Elishtar** [comment]
Done

**Comment by ArtDu** [comment]
@Elishtar I fixed like I wanted it. If you don't mind I'll merge it

**Review by ArtDu — APPROVED** [review]

**Pull request merged**